### ISSUE TITLE
ngx-jodit-pro: Icons are not being rendered

### ISSUE THREAD

**Issue by vitorlopez** [julianpoemp/ngx-jodit#8]
- What version of the package? 3.0.0.beta.1
- What Angular version are you using? Angular 16

So I'm using ngx-jodit-pro in an app with Angular 16 + esbuild and I noticed the icons aren't being rendered correctly:
![image](https://github.com/julianpoemp/ngx-jodit/assets/1822793/8fc78e8f-dea2-4919-a3d8-e29e8c79e641)

I'm also having problems making the plugins to work, for example the page-break plugin. This is my angular.json:

```
"styles": [
                    (...)
                    "./node_modules/jodit-pro/es2021/jodit.fat.min.css"
                ],
                "scripts": [
                    "./node_modules/jodit-pro/es2021/jodit.js",
                    "./node_modules/jodit-pro/es2021/plugins/page-break/page-break.js"
                ]
```

I think I'm importing everything it needs but for some reason still not rendering the icons and the plugins are not working. Let me know if you need more info to reproduce it. But I have a very basic example in my code.

```html
// options: "buttons": "bold,italic,underline", 
<ngx-jodit-pro
                            *ngIf="value"
                            [(value)]="value"
                            [options]="options"
                        />
```

Am I missing something in this setup?

**Comment by julianpoemp** [comment]
thank you for reporting this issue! I solved it by reverting the commit related to typings for options. I thought I could use typings from the Jodit package but this caused the problem.

I just released 3.0.0-beta.2 which should fix this issue 😄

**Comment by vitorlopez** [comment]
That fixed my issue, thanks again @julianpoemp !
